ex1/index.html

```diff
@@ -16,11 +16,11 @@ <h2>This is my blue button</h2>
     id="demo">
     Hit Me!!!
   </button>
-  <img src="noajz.jpg" alt="Smiley face" height="300" width="300" id="noaIMG">
   </div>
-  <div class = "newButton">
+  <img src="noajz.jpg" alt="Smiley face" class="hide-element" height="300" width="300" id="noa-img">
+  <div>
     <button type="button"
-      class="centerOfScreen"
+      class="button-to-restart-array"
```

Comment:
we already know this is a button (the class in a "button" element) so need to mention this in the class.  please change to: restart-array (it's ok that it's similar to the function name)